tools/analyzer_plugin/test/test_util.dart

```diff
@@ -152,3 +151,40 @@ List<AnalysisError> _filterIgnored(List<AnalysisError> errors, IgnoreInfo ignore
 
   return errors.where((e) => !isIgnored(e)).toList();
 }
+
+/// Returns [expression] parsed as AST.
+///
+/// This is accomplished it by including the [expression] as a statement within a wrapper function
+/// with any necessary [imports] at the top of the source. As a result, the offset of the
+/// returned expression will not be 0.
+///
+/// To return resolved AST, set [isResolved] to true.
+///
+/// Returns null if [expression] is not an [InvocationExpression].
```

Comment:
Is there a use-case for this returning null? Otherwise I think we should just throw instead of failing silently